addons/mail/static/src/new/rtc/call_action_list.xml

```diff
@@ -3,16 +3,16 @@
 
     <t t-name="mail.call_action_list" owl="1">
         <div class="o-mail-call-action-list d-flex justify-content-between" t-attf-class="{{ className }}" t-ref="root">
-            <div class="o-mail-call-action-list-buttons d-flex align-items-center flex-wrap">
+            <div class="d-flex align-items-center flex-wrap">
                 <t t-if="isOfActiveCall and rtc.state.selfSession">
                     <t t-if="rtc.state?.selfSession.isMute" t-set="micText">Unmute</t>
                     <t t-else="" t-set="micText">Mute</t>
-                    <button class="o-mail-call-action-list-button btn d-flex m-1 border-0 rounded-circle shadow-none opacity-100 opacity-75-hover"
-                        t-att-class="{ 'o-isActive': !rtc.state.selfSession.isMute, 'o-isSmall p-2': isSmall, 'p-3': !isSmall }"
```

Comment:
Shouldn't removal of active class on these buttons replaced by something else?

Reply:
The icon changes based on `isMute`. The `isActive` is outdated (was certainly used for test assertion and styling in the past)